packages/misc/src/lib/misc.ts

```diff
@@ -1,3 +1,4 @@
+import { LitNodeClientConfig } from '@lit-protocol/types';
```

Comment:
Is there a reason we did not also scope the `loggerManager` in this pr? I can handle this with my follow up but want to confirm.

Reply:
Just wanted to avoid creating a big amount of conflicts with your changes. I can save it also in a variable in `misc` and import it into core returned from `bootstrapLogManager`. Or as you prefer, let me know 😃

Reply:
I think this makes sense, Ill merge this into my branch where I am working on a couple follow ups